tests/recommender.test.js

```
@@ -0,0 +1,67 @@
+const test = require('node:test');
```

ESLint

[error] 'require' is not defined.

(no-undef)